src/command-line/install.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+const colors = require("colors/safe");
+const child = require("child_process");
+const program = require("commander");
+const path = require("path");
+const packageJson = require("package-json");
+const fs = require("fs");
+const Helper = require("../helper");
+const Utils = require("./utils");
+
+program
+	.command("install <package>")
+	.description("Install a package")
+	.on("--help", Utils.extraHelp)
+	.action(function(packageName) {
+		if (!fs.existsSync(Helper.CONFIG_PATH)) {
+			log.error(`${Helper.CONFIG_PATH} does not exist.`);
+			return;
+		}
+
+		packageJson(packageName, {
+			fullMetadata: true
+		}).then((json) => {
+			if (!("lounge" in json)) {
+				log.error(`${colors.red(packageName)} does not have The Lounge metadata.`);
+
+				process.exit(1);
+			}
+
+			const npm = child.spawn(
+				path.sep === "\\" ? "npm.cmd" : "npm",
+				[
+					"install",
+					"--global",
```

Comment:
Hmm, doesn't `--global` break this, as discussed in #1266 ?

Reply:
It worked in my testing.